lib/utils/schema.js

```diff
@@ -20,16 +20,30 @@ module.exports = function () {
               data: isArray ? Joi.array().items(itemSchema) : Joi.object().keys(itemSchema)
             });
         });
+        let dataType = Joi.string().valid(schema.type).required();
+        if (patch) {
```

Comment:
Both 'id' and 'type' are required members in the PATCH payload, the code below makes 'type' optional and leaves 'id' optional as well. 

> The PATCH request MUST include a single resource object as primary data. The resource object MUST contain type and id members.